srcs/mm.c

```diff
@@ -39,4 +39,5 @@ void mem_init()
 	}
 	// test();
 	kheap = init_heap(0x10000, 0xFFFFE000, 0, 1);
+	init_vmpool(VMALLOC_START, VMALLOC_END);
 }
```

Comment:
```suggestion
uint32_t tmp2 = vmalloc(0x10000);
}
```
<img width="495" alt="image" src="https://github.com/4kfs2/kfs/assets/62678380/046b607e-d231-47be-83d2-69def962062c">

<img width="419" alt="image" src="https://github.com/4kfs2/kfs/assets/62678380/56381ef2-44b4-4789-b76d-eaf72e9ca83b">

0x10000을 요청했는데 0x10000 만큼의 페이지 생성이 안됩니다

Reply:
```suggestion
uint32_t tmp2 = vmalloc(0x10000);
vfree(tmp2);
}
```
재부팅이 되는데 이것도 좀 확인해주세요 ㅠ